code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -60,6 +60,11 @@
 	excited = TRUE
 
 	var/atom/target = holding || get_turf(src)
```

Comment:
don't do || get_turf here if that case is never going to be respected

Reply:
Right, I forgot to add a scrub(target_turf.return_air()) before the for loop

Reply:
@Ghilker ?